astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -169,7 +180,7 @@ def execute(self, context: Context) -> None:
 
     def execute_complete(
         self, context: Context, event: Optional[Dict[str, Union[str, List[str]]]] = None
-    ) -> None:
+    ) -> Any:
```

Comment:
Do we need `Any `? can we  not specify the exact ones?

Reply:
It can be a list of Dict, None, tuples, or list of tuples, I initially assigned it as the union of list, tuple still it was complaining